### ISSUE TITLE
[FEATURE] Docker Container

### ISSUE THREAD

**Issue by rg9400** [Strip3s/PhoenixBot#110]
# Summary

The project uses venv to package and control python dependencies, but it also relies on Chrome. Since Chrome versions can vary so widely and can create issues, a container could help peg dependencies properly in an isolated environment. It would also make it easier to update as changes are added and install. 

# MVP Features

A Dockerfile that allows the user to mount a config file. It would be built on a python base image, then add any python dependencies, and then pull in or use CDP in some fashion.

The complexities are that the GUI is built as a desktop application and not a webapp, so accessing it once running a container would require some sort of RDP/VNC which is complicated but doable (LSIO does this with their calibre container for example). Though if everything can be configured via a file, then the user could just populate that and mount it instead.

**Comment by hypercoyote** [comment]
Yeah, I think it's awesome someone picked this up, but the documentation is pretty weak.  I can't get it to work in my environment and I don't really understand why.  The interface loads, but I can't add any tasks after filling in all the details of one.  So that might be a Chrome issue, but who knows 🤷🏻‍♂️🤷🏻‍♂️
